.vscode/extensions.json

```diff
@@ -4,6 +4,7 @@
     "charliermarsh.ruff",
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
+    "tamasfe.even-better-toml",
```

Comment:
We still need this for syntax highlighting.

Reply:
Should be in alphabetical order